### PR TITLE
Added URL option to fetch

### PR DESCRIPTION
**Pull request by wifijt**
Added ability to specify a URL to fetch from vs the URL pyportal was initialized from

### CODE REVIEW

**Comment by brentru** [comment]
@wifijt  Please fix these syntax issues so we can merge in this pull request: https://travis-ci.com/adafruit/Adafruit_CircuitPython_PyPortal/builds/109194515#L293

Thanks!

File: adafruit_pyportal.py [hunk 1]
```diff
@@ -41,6 +41,12 @@
   https://github.com/adafruit/circuitpython/releases
 
 * Adafruit's Bus Device library: https://github.com/adafruit/Adafruit_CircuitPython_BusDevice
+"""
+"""
+04/19/2019
+John Turner
+Added ability to call the fetch object with a URL fetch(URL) this allows the URL to be updated once pyportal is initialized
```
Comment (brentru) [inline adafruit_pyportal.py]: Please remove changenotes from the file, they'll be reflected in the release notes.

Comment (wifijt) [reply adafruit_pyportal.py]: Fixed

**Comment by brentru** [comment]
Thanks for removing the comments!

**Pull request merged**